ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -486,19 +487,31 @@ object Config {
 
         opt[Unit]("early-access")
           .optional()
-          .action((_, c) => c.copy(engineMode = EngineMode.EarlyAccess))
+          .action((_, c) => c.copy(allowedLanguageVersions = LanguageVersion.EarlyAccessVersions))
           .text(
             "Enable preview version of the next Daml-LF language. Should not be used in production."
           )
 
         opt[Unit]("daml-lf-dev-mode-unsafe")
           .optional()
           .hidden()
-          .action((_, c) => c.copy(engineMode = EngineMode.Dev))
+          .action((_, c) => c.copy(allowedLanguageVersions = LanguageVersion.DevVersions))
           .text(
             "Enable the development version of the Daml-LF language. Highly unstable. Should not be used in production."
           )
 
+        opt[Unit]("daml-lf-min-version-1.14")
+          .optional()
+          .hidden()
+          .action((_, c) =>
+            c.copy(allowedLanguageVersions =
+              c.allowedLanguageVersions.copy(min = LanguageVersion.v1_14)
+            )
+          )
+          .text(
+            "Set minimum LF version for unstable packages to 1.14. Should not be used in production."
+          )
```

Comment:
@mziolekda @miklos-da 
This PR adds a feature needed by canton. However we need a way to test it through the full stack. 
Are you fine we add this flag to be able to use the `ledger-on-memory` to test the feature ?

Reply:
Yes, with the above small change. Though, it would be preferred to parse the actual version so that we don't have to modify this command-line parameter for the next Daml-LF version.

Reply:
> Though, it would preferred to parse the actual version so that we don't have to modify this command-line parameter for the next Daml-LF version.

I agree that it would be preferable to parse the actual min version, but note that there's no plan currently for the 1.14 minimum to change in the future, even when the next LF version is released.